src/Playwright.Tests/BrowserContextRouteTests.cs

```diff
@@ -150,5 +182,25 @@ public async Task ShouldFallBackToContextRoute()
             var response = await page.GotoAsync(Server.EmptyPage);
             Assert.AreEqual("context", await response.TextAsync());
         }
+
+        [PlaywrightTest("browsercontext-route.spec.ts", "should throw on invalid route url")]
```

Comment:
nit: as this test is specific to C#, you don't need to annotate it with the link to the upstream test.